### DISCUSSION TITLE
Line break on the homepage

### DISCUSSION THREAD

**Discussion by emddummi** [tteck/Proxmox#2343] [General]
![Screenshot_2024-01-15-07-57-30-319_com android chrome](https://github.com/tteck/Proxmox/assets/1787935/649986b8-fec2-492c-a0aa-49ae6703685c)

Recently I have these many line breaks on the homepage.  Is it wanted like that?

**Comment by tteck** [comment]
The website is currently undergoing improvements with a focus on desktop browsers, and there might be some limitations or less optimal performance on mobile browsers. I'm working towards finding a balance or "happy medium" between the two platforms.

**Reply by tteck** [reply]
Some of the improvements:
1) Correcting pointers.
2) Changing hover colors to better indicate categories/items.
3) Opening links in a new tab.
4) Enhancing the copy button to better indicate that the command has been successfully copied.
5) Adjusting margins (this is the reason mobile browsers look different).

Not website related, but worth mentioning: The logo in the notes in a newly created LXC is a link to the website, and it will now open in a new tab.
